sdk/template/Azure.Template/src/Models/SecretBundle.Serialization.cs

```diff
@@ -77,5 +80,55 @@ internal static SecretBundle FromResponse(Response response)
             using var document = JsonDocument.Parse(response.Content);
             return DeserializeSecretBundle(document.RootElement);
         }
+
+        /// <summary>
+        /// TBD.
+        /// </summary>
+        /// <param name="secretBundle"></param>
+        public static implicit operator RequestContent(SecretBundle secretBundle)
+        {
+            if (secretBundle == null)
+            {
+                return null;
+            }
+
+            return RequestContentExtensions.Create(secretBundle, ModelReaderWriterOptions.DefaultWireOptions);
```

Comment:
RequestContent implicit cast operator calls a "preview Azure.Core" API in the Azure.Core namespace but that lives in the Azure.Core.Experimental package instead of Azure.Core.